feature_engine/creation/combine_with_reference_feature.py

```diff
@@ -158,11 +162,15 @@ def __init__(
         if missing_values not in ["raise", "ignore"]:
             raise ValueError("missing_values takes only values 'raise' or 'ignore'")
 
+        if not isinstance(drop_original, bool):
+            raise TypeError("drop_original takes only boolean values True and False.")
```

Comment:
can we add to the error message f"Got {drop_original} instead."

I am trying to roll out that way of displaying errors across transformers. It is very sklearn like :p

Reply:
Sure